modules/nf-core/sgdemux/nextflow.config

```diff
@@ -0,0 +1,46 @@
+params {
```

Comment:
I don't think we need this file.  I don't see it in `fastqc` or `bcl2fastq` for example.  Thoughts?

Reply:
You are correct, `fastqc` nor `bcl2fastq` have `nextflow.config` files I will remove this file for sgdemux.